src/Tools/NodeDocumentationMarkdownGenerator/Commands/RenameCommand.cs

```diff
@@ -0,0 +1,90 @@
+using System;
+using System.IO;
+using System.Linq;
+using NodeDocumentationMarkdownGenerator.Verbs;
+
+namespace NodeDocumentationMarkdownGenerator.Commands
+{
+    internal static class RenameCommand
+    {
+        internal static void HandleRename(RenameOptions opts)
+        {
+            if (opts.InputMdFile is null && opts.InputMdDirectory != null)
+            {
+                RenameDirectory(opts.InputMdDirectory, opts.MaxLength);
```

Comment:
Why is there a `maxlength `arg used here but not for `RenameFile`?

Reply:
My thinking was that if you rename a single file then you do it for a reason (it is too long) and then then length check is not needed.